packages/react/src/app-layout/AppLayoutSidebarNav.tsx

```diff
@@ -270,3 +347,15 @@ function AppLayoutSidebarNavDivider({
 		</Box>
 	);
 }
+
+function hasSubLevelActiveItem(
+	items: NavItem[] | undefined,
+	bestMatch: string | undefined
+): boolean {
+	if (!(items?.length && bestMatch)) return false;
+	return items.some(
+		(item) =>
+			('href' in item && item.href === bestMatch) ||
```

Comment:
Should make this consistent with side nav with both having the same syntax

Reply:
Cool cool, I will remove the `return true` stuff from `SideNav`. It won't be exact because we have to support buttons without `href`s in AppLayout.

- [ ] make SideNav `hasSubLevelActiveItem` match AppLayout's